src/inference/include/ie/ie_plugin_config.hpp

```diff
@@ -463,6 +463,35 @@ DECLARE_CONFIG_KEY(CACHE_DIR);
  */
 DECLARE_CONFIG_KEY(FORCE_TBB_TERMINATE);
 
+/**
+ * @brief Number of streams in Performance-core(big core) in HYBRID_AWARE and throughput
+ *        The value is calculated in loadExeNetwork and is used in binding CPU cores.
+ */
+DECLARE_CONFIG_KEY(BIG_CORE_STREAMS);
```

Comment:
Why you are defining these as publicly available configuration keys? Are we expect OV user to tune all of them?